java/src/IceGridGUI/build.gradle

```diff
@@ -70,9 +50,7 @@ jar {
 
 project.ext.libJars = []
 
-project.ext.jarBuilder = icegridguiProguard.toBoolean() ? "proguard-jar.gradle" : "plain-jar.gradle"
-
-apply from: jarBuilder
+apply from: "proguard-jar.gradle"
```

Comment:
We don't need to support plain jar, we are always building the proguard version.